src/cascade/dismod/db/wrapper.py

```diff
@@ -187,15 +181,40 @@ def __dir__(self):
         attributes.extend(super().__dir__())
         return attributes
 
+    def update_table_columns(self, table_definition, table):
+        new_columns = table.columns.difference(table_definition.c.keys())
+        new_columns = {c: table.dtypes[c] for c in new_columns}
+
+        bad_column_names = [c for c in new_columns.keys() if not c.startswith("x_")]
+        if bad_column_names:
+            raise ValueError("Covariate column names must start with 'x_'")
```

Comment:
Add the list of bad column names to the error message.

Reply:
Sure